### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by 26th-nov** [skills/.github#335] [General]
Hey, I'm new here.

Open to suggestions on "**what to do?**" and "**where to start?"**

Thanks!

**Comment by sinsukehlab** [comment]
Select courses you like and follow the instructions: https://skills.github.com/

**Comment by AlliumGalore** [comment]
Hi, there's a good opportunity to have an intro on MS Learn.

**Comment by MsBattles** [comment]
I started the "https://skills.github.com/" and I got to Step 2, and completed it, the course has not advanced after refreshing... Any suggestions?
